x11-libs/libXcursor/libXcursor-1.2.0.ebuild

```diff
@@ -17,6 +17,6 @@ RDEPEND=">=x11-libs/libXrender-0.9.8[${MULTILIB_USEDEP}]
 DEPEND="${RDEPEND}"
 
 XORG_CONFIGURE_OPTIONS=(
-	--with-icondir=/usr/share/cursors/xorg-x11
+	--with-icondir="${EPREFIX}"/usr/share/cursors/xorg-x11
```

Comment:
If you haven't already, test installation to see if it already respected prefix? Also, this isn't in src_configure unlike the others?

Reply:
yup, because there's no USE dependencies?

Reply:
oh, duh. I hadn't clocked the logic for it :|